CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.4.0
+ - Added oneview_scope resource
```

Comment:
Sorry, I forgot to mention that it would be good adding here too that it was added on API300.